pystan/tests/test_extra_compile_args.py

```diff
@@ -39,89 +39,3 @@ def test_extra_compile_args_failure(self):
         with assertRaisesRegex(distutils.errors.CompileError, 'failed with exit status'):
             pystan.StanModel(model_code=model_code, model_name="normal1",
                              extra_compile_args=extra_compile_args)
-
-    def test_threading_support(self):
```

Comment:
Can we add

    import unittest

    @unittest.skip("temporarily disabled")
    def test_threading_support

which causes it not to be run with other tests, but let's us test is locally?

https://docs.python.org/2/library/unittest.html#unittest.skip